diff --git a/src/Entity/GallyConfiguration.php b/src/Entity/GallyConfiguration.php
--- a/src/Entity/GallyConfiguration.php
+++ b/src/Entity/GallyConfiguration.php	(date 1700494097000)
@@ -6,7 +6,7 @@

 use Sylius\Component\Resource\Model\ResourceInterface;

-final class GallyConfiguration implements ResourceInterface, GallyConfigurationInterface
+class GallyConfiguration implements ResourceInterface, GallyConfigurationInterface
 {
     protected ?int $id;
     protected string $baseUrl;
